src/language-css/parser-postcss.js

```diff
@@ -266,6 +266,14 @@ function parseMediaQuery(params) {
 
   let result = null;
 
+  // bail on SCSS comments inside the query
+  if (/\/\//.test(params)) {
+    return {
+      type: "selector-unknown",
+      value: params.trim(),
+    };
+  }
```

Comment:
Bad idea, we should handle comments, not as `selector-unknown`, it breaks whole formatting

Reply:
The thing is that's [the strategy used for selectors](https://github.com/prettier/prettier/blob/master/src/language-css/parser-postcss.js#L233), so I just copied that. But of course a proper solution would be better; I'll look into that.